src/tcgwars/logic/impl/gen3/PopSeries2.groovy

```diff
@@ -313,7 +375,7 @@ public enum PopSeries2 implements LogicCardInfo {
           energyCost C
           attackRequirement {}
           onAttack {
-
+            // This also does nothing
```

Comment:
This needs an attackRequirement. Probably `assert false : "This attack does nothing"`